gradle.properties

```diff
@@ -25,3 +25,5 @@ POM_DEVELOPER_ID=square
 POM_DEVELOPER_NAME=Square, Inc.
 POM_DEVELOPER_URL=https://github.com/square/
 SONATYPE_STAGING_PROFILE=com.squareup
+
+kotlin.js.compiler=ir
```

Comment:
Do we have a link to a support article describing why this is the best?

Reply:
https://kotlinlang.org/docs/js-ir-compiler.html

_Rather than directly generating JavaScript code from Kotlin source code, the Kotlin/JS IR compiler backend leverages a new approach. Kotlin source code is first transformed into a [Kotlin intermediate representation (IR)](https://kotlinlang.org/docs/whatsnew14.html#unified-backends-and-extensibility), which is subsequently compiled into JavaScript. For Kotlin/JS, this enables aggressive optimizations, and allows improvements on pain points that were present in the previous compiler, such as generated code size (through dead code elimination), and JavaScript and TypeScript ecosystem interoperability, to name some examples._

Reply:
Thanks! I should have been clearer though with the comment. I meant can you include that link as a comment in the .properties file so we have reference in the future

Reply:
would just adding the link suffice? or do we prefer also a snippet of what this is?

Reply:
added a bit of comment explaining the choice and the link